client/src/banners/banner-utils.ts

```diff
@@ -0,0 +1,44 @@
+// Set a localStorage key with a timestamp the specified number of
+// days into the future. When the user dismisses a banner we use this
+// to prevent the redisplay of the banner for a while.
+export function setEmbargoed(id: string, days: number) {
+  try {
+    let key = `banner.${id}.embargoed_until`;
```

Comment:
I'd use `const` if possible:
```suggestion
    const key = `banner.${id}.embargoed_until`;
```